tests/layer_tests/jax_tests/test_argmax.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+from jax import lax
+from jax import numpy as jnp
+
+from jax_layer_test_class import JaxLayerTest
+
+
+def generate_shape_axis_pairs(input_shapes):
+    test_data = []
+    for shape in input_shapes:
+        rank = len(shape)
+        # Only [0, rank - 1] are valid axes for lax.argmax
+        valid_axes = range(0, rank)
+        test_data.extend([{'input_shape': shape, 'axis': axis}
+                         for axis in valid_axes])
+    return test_data
+
+
+class TestArgmax(JaxLayerTest):
+    def _prepare_input(self):
+        inp = jnp.array(np.random.rand(*self.input_shape).astype(np.float32))
```

Comment:
please check with different input type. Also add a case when max element is duplicating in a tensor along given axis.

Reply:
Fixed in [c85dba](https://github.com/openvinotoolkit/openvino/pull/26671/commits/c85dba315525b6bd69475d3acf33b1598926ce08)